tests/model/stepfunctions/test_state_machine_generator.py

```diff
@@ -73,7 +69,7 @@ def test_state_machine_both_role_and_policies(self):
             StateMachineGenerator(**self.kwargs).to_cloudformation()
         self.assertEqual(
             error.exception.message,
-            "Resource with id [StateMachineId] is invalid. Specify either 'Role' or 'Policies' property and not both.",
+            "Resource with id [StateMachineId] is invalid. Specify 'Role' or 'Policies' or neither property.",
```

Comment:
I'd suggest either:
- assert only error class if it's enough
- make an error message a constant where you raise it ie `SFN_INVALID_PROPERTY_BLAH_BLAH = "Our error message"` and import it here
Otherwise, it's additional friction to maintain unit tests up-to-date without no value in this case